opt-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+services:
+
+  postgres_db:
+    image: postgres
+    restart: always
+
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=school
+      - POSTGRES_DB=school
+
+    volumes:
+      - ../../data/postgres:/var/lib/postgresql/data
+      - ../../data/dump:/db-dump
+
+    ports:
+      - "5432:5432"
+
+    networks:
+      - docker-practice-net
+
+
+  redis:
+    image: "redis:alpine"
+    command: redis-server
```

Comment:
I tried to bring up Redis just via `docker run redis:alpine` and this is up and running and: `Ready to accept connections`
Do we need to redefine Redis launch via `redis-server` command?

Reply:
When I ran the docker-compose up command, everything worked fine, so you don't need to add anything.

Reply:
I agree that `docker-compose up` command should does everything what project needs. 

But I'm talking about a little bit different things:
- I tried to use the same image
- BUT, I did not set `redis-server` command in my `docker run redis:alpine`, I did not overwrite its environment variables and so on.
- It means that probably those overwrites are not needed :) 

I think it's more clear now. Let me know if you have anything.